Dockerfile

```diff
@@ -0,0 +1,18 @@
+# This is a development Dockerfile. For versioned Dockerfiles see:
+# https://github.com/praekeltfoundation/docker-seed
+FROM praekeltfoundation/django-bootstrap:py2.7-jessie
```

Comment:
Do you need to use `jessie`? If things work without the `-jessie` I'd prefer that as Debian Jessie is kind of out-of-date these days.

Reply:
it seems like jessie is the most recent one on this list: https://cloud.docker.com/u/praekeltfoundation/repository/docker/praekeltfoundation/django-bootstrap

Reply:
We still build Jessie images because they're needed in some cases. But unless you need a specific Debian version I would recommend without the `-jessie` which gives you the latest version (which right now is `stretch`).

Reply:
Probably better to use the least specific base image you can. If it turns out an update to stretch or something breaks stuff, we can decide whether to fix whatever's breaking or explicitly build on jessie.

Reply:
Oki, I removed the version pin